### PR TITLE
Updating Vagrant section to MAMP/WAMP/VVV

### PR DESCRIPTION
**Pull request by rclations**
## Changes

- Updated the vagrant setup documentation to recommend MAMP, WAMP, and VVV, for simpler on boarding for users of all types.
- Updated vagrant setup documentation to direct users to the Largo documentation (instead of deploy-tools), centralizing the documentation on Vagrant setup.

## Why?

We needed a simpler option for new users to get started with development, without having to setup a custom Vagrant build. MAMP & WAMP offer a simple alternative, while VVV streamlines our existing Vagrant process for advanced users and takes care of a lot of the inherent bugs.

*Benefits of VVV*
- Large user base, with an active community and continuing feature development.
- Simpler setup
- Less bugs, and more community support for any bugs that we may find
- Additional tools specifically for WordPress development

In addition, by adopting standard tools widely used in the community, we grow our ability to attract new developers to contribute to our projects.

This would fix issues #41 & #47, by removing the functionality they address in favor of using VVV.

## Related

https://github.com/INN/deploy-tools/pull/48

### CODE REVIEW

**Comment by rclations** [comment]
Files that need updates as a part of this PR
- [X] https://github.com/INN/docs/blob/master/staffing/onboarding/os-x-setup.md
- [ ] https://github.com/INN/docs/blob/master/projects/largo/umbrella-setup.md
- [ ] https://github.com/INN/docs/blob/master/projects/largo/write-the-docs.md
- [ ] https://github.com/INN/docs/blob/master/checklists/child-themes.md

File: staffing/onboarding/vvv-setup.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  v.memory = 2048
+  v.cpus = 4
+end
+```
+
+- That created a new file, `Customfile` which increases the memory limit on your Vagrant box, as [recommended in the VVV documentation](https://github.com/Varying-Vagrant-Vagrants/VVV/wiki/Customising-your-Vagrant's-attributes-and-parameters).
+
+## Setup a Largo Blueprint for VV
+
+One useful feature of VV is that it allows you to create "blueprints" for site configurations. Below, we'll setup a basic blueprint for Largo sites.
```
Comment (benlk) [inline staffing/onboarding/vvv-setup.md]: Will we need to create new blueprints for each of the Largo umbrellas?

Comment (rclations) [inline staffing/onboarding/vvv-setup.md]: I don't believe so - the general `vv create` command will walk through site setup options, where you can specify a mysql database and external repo to use for `wp-content`, which I think is probably the way to go.

We'll need to modify our bitbucket repo a bit so things match up better, but that should be about it.

**Comment by rclations** [comment]
I think this is just about ready for review - the VVV pieces should be all set, there's just a few other related things I noticed

1. I have removing the Vagrantfile as a step in here, but if we want to just remove that from bitbucket when we're ready, that would be cleaner

2. using fabric to download the mysql database doesn't appear to work (at least in my testing). @benlk sent me some instructions for using curl to do this, and we should probably make that the standard.

3. The search/replace script is a good one, but also fails for me because of the database size. Running a terminal command on the downloaded mysql file *before* importing it may be a better way to go. I was thinking something like `sed -n 's/largoproject.wpengine.com/largo-umbrella.dev/g' mysql-edited.sql`, but it didn't work on my first pass, so I want to debug that and get something really dependable setup there.

File: projects/largo/umbrella-setup.md [hunk 2]
```diff
@@ -20,7 +20,78 @@ Most of this stuff works from your command line, which you get to through the de
...
+
+Remove default wp-content folder
+```
+rm -rf wp-content
+```
+
+Run the following commands to initialize a new git repository and sync it with the largo-umbrella remote. This is similar to cloning a git repository, but lets us work with the files and folders we already have setup.
+```
+git init
+git remote add origin git@bitbucket.org:rclations/largo-umbrella.git
```
Comment (benlk) [inline projects/largo/umbrella-setup.md]: This repo is not currently public; will we be making it public or merging it into the `largoproject` version?

File: projects/largo/umbrella-setup.md [hunk 3]
```diff
@@ -20,7 +20,78 @@ Most of this stuff works from your command line, which you get to through the de
...
+Install as subdomain or subdirectory? : | subdomain 
+Git repo to clone as wp-content (leave blank to skip): | *hit [Enter]*
+Local SQL file to import for database (leave blank to skip): | *This directory must be an absolute path, so the easiest thing to do is to drag your mysql file into your terminal window here and the absolute filepath with fill itself in.*
+Remove default themes and plugins? (y/N): | N 
+Add sample content to site (y/N): | N 
+Enable WP_DEBUG and WP_DEBUG_LOG (y/N): | y
+ 
+This will take a few minutes to build your new site and load in the database, so be patient.
+
+## 3. Add the Largo Umbrella files to your new site
```
Comment (benlk) [inline projects/largo/umbrella-setup.md]: Regarding this whole section: If one already has an existing umbrella repo under the old scheme, is it possible to link the old site's folder to the vvv htdocs folder using `ln -s` or similar, or is that approach inadvisable?

Comment (rclations) [inline projects/largo/umbrella-setup.md]: Probably easier long-term to just rebuild using this method if you're going to use vvv - that method may work, but it also might have unexpected results.

One option is that you could build a site using the setup instructions, then just replace the repo files with your original files if you have things setup there you don't want to lose

Comment (benlk) [inline projects/largo/umbrella-setup.md]: Both your options make more sense than `ln -s /old/dir /htdocs`, at the expense of disk space. Copying the assets directory from an existing umbrella is probably the way to go.

File: projects/largo/umbrella-setup.md [hunk 4]
```diff
@@ -187,27 +211,7 @@ You can name the snapshot anything you want, and I would recommend describing it
...
-    * Password: `root`
-    * Database Host: `localhost`
-    * Table Prefix: `wp_`
-
-4. Go to the next screen, where you have only one button to click, `Run the install`. Click that, and it will create the file `wp-config.php` in your `largo-umbrella` folder.
-
-5. You may then see the message "Already Installed" because you already loaded the WordPress database in a previous step.
-
-## 13. Changing largoproject.wpengine.com to vagrant.dev.
+## 12. Changing largoproject.wpengine.com to vagrant.dev.
```
Comment (benlk) [inline projects/largo/umbrella-setup.md]: Should this section also include the `wp` options seen here: https://github.com/INN/docs/blob/master/projects/largo/database-reload.md#4-replace-primary-wp_-tables-with-local-environment-urls

Comment (rclations) [inline projects/largo/umbrella-setup.md]: yeah, definitely - I'll add that in

Comment (rclations) [inline projects/largo/umbrella-setup.md]: updated in c9c02e3

File: projects/largo/umbrella-setup.md [hunk 5]
```diff
@@ -20,7 +20,78 @@ Most of this stuff works from your command line, which you get to through the de
...
+vv create
+```
+
+Follow the prompts, entering the information below, or hitting [Enter] to skip.
+
+Prompt | Text to enter 
+------------ | -------------
+Name of new site directory: | largo-umbrella 
+Blueprint to use (leave blank for none or use largo): | *hit [Enter]* 
+Blueprint to use (leave blank for none or use largo): | *hit [Enter]*
```
Comment (benlk) [inline projects/largo/umbrella-setup.md]: doubled line here

**Pull request merged**